portfolio/src/main/webapp/script.js

```diff
@@ -12,17 +12,13 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/**
- * Adds a random greeting to the page.
-*/
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
-
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
-
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
+function getMessages() {
+  fetch('/data').then(response => response.json()).then((messages) => {
+    const messagesElement = document.getElementById('messages-container');
+    messagesElement.innerHTML = "";
+    console.log(messages);
```

Comment:
general comment about logging: it's always a good idea to add some context to the log message to help understand what's being logged. Seeing a random log line that says ["blah"] is less helpful than "Messages received from server: ["blah"]".